news/2024/2024-11-28-osu-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,135 @@
+| **Canada** ::{ flag=CA }:: | **6** | 5 | ::{ flag=CN }:: China |
+| Australia ::{ flag=AU }:: | 4 | **6** | ::{ flag=US }:: **United States** |
+| **South Korea** ::{ flag=KR }:: | **6** | 0 | ::{ flag=DE }:: Germany |
+| Hong Kong ::{ flag=HK }:: | 3 | **6** | ::{ flag=CL }:: **Chile** |
+| **Russian Federation** ::{ flag=RU }:: | **6** | 2 | ::{ flag=CL }:: Chile |
+| Canada ::{ flag=CA }:: | 5 | **6** | ::{ flag=GB }:: **United Kingdom** |
+
+This semifinals weekend was historic; teams rose and fell like kingdoms, and the long-standing empires once more laid claim to their promised land. Glorious leaders commanded their troops through a weekend of fierce battles where the top 12 became the top 6 and where even mother nature stepped onto the battlefield.
+
+The first round of losers' bracket looked to be typical. ::{ flag=RU }:: Russia took down ::{ flag=PL }:: Poland 6–1, and ::{ flag=CL }:: Chile defeated ::{ flag=HK }:: Hong Kong in a tightly-contested 6–3 match, winning one map by only 107 points! This marks ::{ flag=PL }:: Poland's fourth top 12 finish in four years and ::{ flag=HK }:: Hong Kong's seventh in eight years.
+
+The resulting potential here would see ::{ flag=RU }:: Russia take down ::{ flag=CL }:: Chile 6–2 in the second round of the losers' bracket. Chile, having made the top 8 twice in a row, are finally solidifying themselves as a top performer after the unfortunate series of events in OWC 2021 and 2022. ::{ flag=RU }:: Russia, on the other hand, have finally broken their semifinals curse; the last time Russia finished in the top 6 was in 2019, and this roster finally made the cut.
+
+On the other side of the losers' bracket, ::{ flag=CA }:: Canada and ::{ flag=CN }:: China fought all the way until tiebreaker, with ::{ flag=CA }:: Canada taking the last map and advancing 6–5 into top 8. ::{ flag=CN }:: China showed once more, after taking both ::{ flag=RU }:: Russia and ::{ flag=CA }:: Canada to tiebreaker, that they are not to be trifled with.
+
+Across from this match was the ::{ flag=PH }:: Philippines vs ::{ flag=GB }:: United Kingdom, where the ::{ flag=PH }:: Philippines gave the UK some trouble, with the ::{ flag=GB }:: UK winning 6–4. The ::{ flag=PH }:: Philippines have now made the top 12 for the second time in OWC, solidifying them as one of the tournament's high performers.
+
+Meanwhile, in the winners' bracket:
+
+- ::{ flag=DE }:: Germany matched up against ::{ flag=KR }:: South Korea in an underdog match. And certainly, ::{ flag=DE }:: Germany were the underdogs; ::{ flag=KR }:: South Korea continued their 16–0 win streak and dropped no points against ::{ flag=DE }:: Germany, making them 22–0 in maps and 4–0 in matches through OWC 2024.
+- ::{ flag=AU }:: Australia and the ::{ flag=US }:: United States faced off in the last match of the weekend, with the ::{ flag=US }:: USA taking the match 6–4 and securing a spot in the winners' finals for the first time since 2021.
```

Comment:
```suggestion
- ::{ flag=AU }:: Australia and the ::{ flag=US }:: United States faced off in the last match of the weekend, with the ::{ flag=US }:: U.S. taking the match 6–4 and securing a spot in the Winners' Finals for the first time since 2021.
```